From: Jason Rhinelander <jason@imaginary.ca>
Date: Thu, 25 Jul 2019 19:10:21 -0300
Subject: Remove ccache

The default pbuilder install already puts ccache in the path, and having
this results in double-ccache invocation which looks odd in the build
log.
---
 CMakeLists.txt | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5ead30f..daf3409 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,11 +1,6 @@
 # Lowest version - android ndk 3.6.0
 cmake_minimum_required(VERSION 3.6.0)
 
-find_program(CCACHE_PROGRAM ccache)
-if(CCACHE_PROGRAM)
-  set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE "${CCACHE_PROGRAM}")
-endif()
-
 set(PROJECT_NAME lokinet)
 project(${PROJECT_NAME} C CXX)
 
